src/ui/reportView/callReport.ts

```diff
@@ -109,54 +146,64 @@ function createCommand(
 }
 
 /**
- * Run the visualize command to generate the report, parse the output to return the python server command and status
+ * Run the visualize command to generate the report, parse the output and return the result
  *
  * @param command - the cargo kani | kani command to run --visualize
- * @returns - A promise of the python command and the status code; Promise<visualizeOutput>
+ * @returns - the result of executing the visualize command and parsing the output
  */
 async function runVisualizeCommand(command: string, harnessName: string): Promise<visualizeOutput> {
 	try {
 		vscode.window.showInformationMessage(`Generating viewer report for ${harnessName}`);
 		const { stdout, stderr } = await execPromise(command);
-		const serveReportCommand: string = await parseReportOutput(stdout);
-		if (serveReportCommand === '') {
-			return { statusCode: 2, serverCommand: stderr };
+		const parseResult = await parseReportOutput(stdout);
```

Comment:
I think there's some value in directly displaying the stderr to the user so they know what's causing the crash. Thoughts?

Reply:
Good catch, thanks! I didn't notice I could pass `stderr` when I refactored this code.